projects/WTI-UI/src/app/modules/clarifications/components/clarifications-page/clarifications-page.component.ts

```diff
@@ -34,7 +34,9 @@ export class ClarificationsPageComponent implements OnInit, OnDestroy {
     this.loadClars();
 
     this._contestService.clarificationsUpdated
-      .pipe(takeUntil(this._unsubscribe))
+      .pipe(
+	delay(300),
```

Comment:
Can you at least document/explain in comments why the `delay(300)` is now needed?